pkg/config/config_test.go

```diff
@@ -279,7 +279,7 @@ func TestCertsConfigCache(t *testing.T) {
 	}
 
 	t.Run("TRISAConfig", createTest(&conf))
-	t.Run("TRPConfig", createTest(&config.TRPConfig{Certs: path, Pool: path}))
+	t.Run("TRPConfig", createTest(&config.TRPConfig{MTLSConfig: config.MTLSConfig{Certs: path, Pool: path}}))
```

Comment:
The use of the embedding required updating all directly initialized `Config`, `TRISAConfig`, and `TRPConfig` structs, which is why these test files were changed.